controller/hashtagcontroller.js

```diff
@@ -11,6 +11,7 @@ const createhashtags = async (req, resp) => {
     resp.status(201).json({ data: data, message: "hashtag create successfully created" })
 }
 
```

Comment:
Use capital case for file names: Eg: HashTagController.js

Reply:
ok